projects/wds/project.yaml

```diff
@@ -0,0 +1,4 @@
+homepage: "https://github.com/intel/wds"
+primary_contact: "<primary_contact_email>"
```

Comment:
Please talk to the maintainers in https://github.com/intel/wds/pull/190 and find someone who will be the primary contact.

Reply:
Minor update: I emailed the developers of wds (reached out to two different people @kanavin and @alexshalamov who have contributed extensively to wds codebase) but did not get a response.

@Dor1s Shall I close this PR or do you happen to have any idea if I can reach out to someone else regarding this?

Reply:
I don't mind to keep it open, maybe @kanavin or @alexshalamov will reply one day